src/components/SignUp/OrganizationInformation.tsx

```diff
@@ -7,7 +7,9 @@ import {
   isValidPhoneNumber, isValidUSState, isValidZipCode, isValidOrgName,
   isValidEIN,
 } from '../../lib/Validations/Validations';
-import CompleteSignupFlow from './CompleteSignupFlow';
```

Comment:
jw why are we removing this?

Reply:
it wasn't used, but i can keep it in if that's what we want?

Reply:
ah i see- sounds good